src/main/scala/com/fulcrumgenomics/bam/OverlappingBasesConsensusCaller.scala

```diff
@@ -241,10 +187,86 @@ object OverlappingBasesConsensusCaller {
   * @param r1CorrectedBases the number of bases modified in R1
   * @param r2CorrectedBases the number of bases modified in R2
   */
-case class CorrectionStats(r1OverlappingBases: Int, r2OverlappingBases: Int, r1CorrectedBases: Int, r2CorrectedBases: Int)
+case class CorrectionStats(overlappingBases: Int = 0, r1CorrectedBases: Int = 0, r2CorrectedBases: Int = 0)
+
+
+/** Trait that entries in [[AgreementStrategy]] will extend. */
+sealed trait AgreementStrategy extends EnumEntry {
+  def qual(qual1: PhredScore, qual2: PhredScore): (PhredScore, PhredScore)
+}
+
+
+/** Enum to represent the strategy to consensus call when both bases agree. */
+object AgreementStrategy extends FgBioEnum[AgreementStrategy] {
+  /** Call the consensus base and return a new base quality that is the sum of the two base qualities. */
+  case object Consensus extends AgreementStrategy {
+    def qual(qual1: PhredScore, qual2: PhredScore): (PhredScore, PhredScore) = {
+      val qual = PhredScore.cap(qual1 + qual2)
+      (qual, qual)
+    }
+  }
+
+  /** Call the consensus base and return a new base quality that is the maximum of the two base qualities. */
+  case object MaxQual extends AgreementStrategy {
+    def qual(qual1: PhredScore, qual2: PhredScore): (PhredScore, PhredScore) = {
+      val qual = PhredScore.cap(Math.max(qual1, qual2))
+      (qual, qual)
+    }
+  }
+
+  /** Leave the bases and base qualities unchanged. */
+  case object PassThrough extends AgreementStrategy {
+    def qual(qual1: PhredScore, qual2: PhredScore): (PhredScore, PhredScore) = (qual1, qual2)
+  }
+
+  override def values: IndexedSeq[AgreementStrategy] = super.findValues
+}
+
+/** Trait that entries in [[DisagreementStrategy]] will extend. */
+sealed trait DisagreementStrategy extends EnumEntry {
+  def baseAndQual(base1: Byte, qual1: PhredScore, base2: Byte, qual2: PhredScore): (Byte, PhredScore, Byte, PhredScore)
+}
+
+/** Enum to represent the strategy to consensus call when both bases disagree.  Masking a base will make the base
+  * an "N" with base quality "2". */
+object DisagreementStrategy extends FgBioEnum[DisagreementStrategy] {
+  import OverlappingBasesConsensusCaller.{NoCall, NoCallQual}
+
+  /** Mask both bases. */
+  case object MaskBoth extends DisagreementStrategy {
+    def baseAndQual(base1: Byte, qual1: PhredScore, base2: Byte, qual2: PhredScore): (Byte, PhredScore, Byte, PhredScore) = {
+      (NoCall, NoCallQual, NoCall, NoCallQual)
+    }
+  }
+
+  /** Mask the base with the lowest base quality, with the other base unchanged.  If the base qualities are the same,
+    * mask both bases. */
+  case object MaskLowerQual extends DisagreementStrategy {
+    def baseAndQual(base1: Byte, qual1: PhredScore, base2: Byte, qual2: PhredScore): (Byte, PhredScore, Byte, PhredScore) = {
+      //                       r1-base  r1-qual                        r2-base  r2-qual
+      if (qual1 > qual2)      (base1,   PhredScore.cap(qual1 - qual2), NoCall,  NoCallQual)
```

Comment:
From the docs I was thinking this would leave the qual1 unchanged, not subtract the qual2 from it.  Thoughts?  I think either way is fine, but the documentation should be made clearer if you leave the code this way.